bitcoin/src/blockdata/transaction.rs

```diff
@@ -1319,7 +1319,8 @@ pub const fn predict_weight_from_slices(
 /// Weight prediction of an individual input.
 ///
 /// This helper type collects information about an input to be used in [`predict_weight`] function.
-/// It can only be created using the [`new`](InputWeightPrediction::new) function.
+/// It can only be created using the [`new`](InputWeightPrediction::new) function or using other
+/// associated constants/methods.
```

Comment:
OT: Do Rust docs typically be careful to differentiate between the word "method" (a function implemented on types) and "function"? I have been trying to do this but its a bit painful.

Reply:
I don't think they're careful. I've never been very careful about this -- I feel like the distinction isn't very useful outside of a particular OO mindset.